docs/topics/Gateway.md

```diff
@@ -561,27 +561,33 @@ When connecting to the gateway as a bot user, guilds that the bot is a part of w
 
 ## Sharding
 
-As apps grow and are added to an increasing number of guilds, some developers may find it necessary to divide portions of their app's operations across multiple processes. As such, the Gateway implements a method of user-controlled guild sharding which allows apps to split events across a number of Gateway connections. Guild sharding is entirely controlled by an app, and requires no state-sharing between separate connections to operate. While all apps *can* enable sharding, it's not necessary for apps in a smaller number of guilds. 
+As apps grow and are added to an increasing number of guilds, some developers may find it necessary to divide portions of their app's operations across multiple processes. As such, the Gateway implements a method of user-controlled guild sharding which allows apps to split events across a number of Gateway sessions. Guild sharding is entirely controlled by an app, and requires no state-sharing between separate sessions to operate. While all apps *can* enable sharding, it's not necessary for apps in a smaller number of guilds.
 
 > warn
-> Each shard can only support a maximum of 2500 guilds, and apps that are in 2500+ guilds *must* enable sharding. 
+> Each shard can only support a maximum of 2500 guilds, and apps that are in 2500+ guilds *must* enable sharding.
 
-To enable sharding on a connection, the app should send the `shard` array in the [Identify](#DOCS_TOPICS_GATEWAY_EVENTS/identify) payload. The first item in this array should be the zero-based integer value of the current shard, while the second represents the total number of shards. DMs will only be sent to shard 0.
+Sessions that would like to only receive events from a subset of guilds should send the `shard` array in the [Identify](#DOCS_TOPICS_GATEWAY_EVENTS/identify) payload. The first item in this array is `shard_id`, the zero-based integer value of the current shard, while the second is `num_shards` and represents the total number of shards.
 
 > info
 > The [Get Gateway Bot](#DOCS_TOPICS_GATEWAY/get-gateway-bot) endpoint provides a recommended number of shards for your app in the `shards` field
 
-To calculate which events will be sent to which shard, the following formula can be used:
+A certain gateway session is only subscribed to events from guilds with a `guild_id` that satisfies the following formula, using the `shard_id` and `num_shards` that the session provided in the [Identify](#DOCS_TOPICS_GATEWAY_EVENTS/identify) event:
 
 ###### Sharding Formula
 
 ```python
-shard_id = (guild_id >> 22) % num_shards
+(guild_id >> 22) % num_shards == shard_id
```

Comment:
I'm not entirely sure why this change is necessary - both say the same thing, and I think the prior version says it a bit more clearly.

Reply:
I think that `a = b + c` is much more clear than `b + c == a`

Reply:
With this PR, the formula is written in the context of - when a shard is connecting to the gateway - iterating through all guilds that a bot is part of and deciding whether the shard should receive events from that guild based on the guilds `guild_id` and the `shard` array with `shard_id` and `num_shards` that the shard provided.
Sure, if you see it as a mathematical equation, a single `=` would suffice, but this is documentation that programmers will be reading to understand how sharding behaves, and when programmers see an expression like that with things that are very clearly variable names they would assume that `=` is an assignment, which it is not.
I agree that writing the single variable on the left-hand side looks cleaner, but as discussed in the thread i mentioned, that would create _some_ ambiguity of whether it was actually meant as a comparison or not.

Reply:
How is this not an assignment? The original wording is much clearer.

Reply:
Although I appreciate the discussion, I would still prefer the original form to remain.

Reply:
Sure, we can keep it as an assignment. It still has to be in the context of a single shard though, as each session has its own (potentially different) value of `num_shards`. Then we can just write with words afterwards that "The session will receive events from every guild that, using the formula above, evaluates to the sessions `shard_id` using the corresponding `num_shards`".
I don't see how adding this level of indirection makes it any clearer though?

Reply:
I think that simply keeping it as is for the time being is fine. I feel the formula is clear enough and one can grok the relationship. 

I've always thought the larger meaning here is: "given a guild, and the number of shards I'm running, which shard does a given guild land on". This answers a practical question, for example if you want to locate the shard that is handling a given guild, so you can look up data from that shard's process. Internally, we do such lookups (albeit with hash rings, but they are logically the same.). This is how the original formula was written. 

The, "which guilds precisely will land on this shard" is a less useful question to answer, as it's not something the developer has control over, and is a fact that one can derive from the original formula as well.